solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -307,6 +323,10 @@ contract RandomBeacon is IRandomBeacon, Ownable {
         address[] groupMembers
     );
 
+    event AuthorizedContract(address contractToAuthorize);
+
+    event UnauthorizedContract(address contractToUnauthorize);
```

Comment:
Looking at other events the convention is that we are putting the verb at the end, so they should be `ContractAuthorized` and `ContractUnauthorized`. If we decide to rename `authorizedContracts` mapping to `authorizedRequesters`, then I would rename them to `RequesterAuthorized` and `RequesterDeauthorized`.

Reply:
It's gone now, replaced by `RequesterAuthorizationUpdated` event